src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/list.html.twig

```diff
@@ -23,117 +23,118 @@
+                <input type="hidden" name="mass_edit_action_sorted_products[]" value="{{ product.id_product }}" />
+                <input type="hidden" name="mass_edit_action_sorted_positions[]" value="{{ product.position }}" />
+            </td>
         {% endif %}
         <td class="text-right">
             <div class="btn-group-action">
 
                 {% set buttons_action = [
-                  {
-                  "href": product.preview_url|default('#'),
-                  "target": "_blank",
-                  "icon": "remove_red_eye",
-                  "label": "Preview"|trans({}, 'Admin.Actions')
-                  }
+                    {
+                        "href": product.preview_url|default('#'),
+                        "target": "_blank",
+                        "icon": "remove_red_eye",
+                        "label": "Preview"|trans({}, 'Admin.Actions')
+                    }
                 ] %}
 
                 {% set buttons_action = buttons_action|merge([
-                  {
-                  "onclick": "unitProductAction(this, 'duplicate');",
-                  "icon": "content_copy",
-                  "label": "Duplicate"|trans({}, 'Admin.Actions')
-                  }
+                    {
+                        "onclick": "unitProductAction(this, 'duplicate');",
+                        "icon": "content_copy",
+                        "label": "Duplicate"|trans({}, 'Admin.Actions')
+                    }
                 ]) %}
 
                 {% set buttons_action = buttons_action|merge([
-                  {
-                  "onclick": "unitProductAction(this, 'delete');",
-                  "icon": "delete",
-                  "label": "Delete"|trans({}, 'Admin.Actions')
-                  }
+                    {
+                        "onclick": "unitProductAction(this, 'delete');",
+                        "icon": "delete",
+                        "label": "Delete"|trans({}, 'Admin.Actions')
+                    }
                 ]) %}
 
                 {% include '@Product/CatalogPage/Forms/form_edit_dropdown.html.twig' with {
-                  'button_id': "product_list_id_" ~ product.id_product ~ "_menu",
-                  'default_item': {
-                    "href": product.url|default('#'),
-                    "icon": "mode_edit"
-                  },
-                  'right': true,
-                  'items': buttons_action
+                    'button_id': "product_list_id_" ~ product.id_product ~ "_menu",
+                    'default_item': {
+                        "href": product.url|default('#'),
+                        "icon": "mode_edit"
+                    },
+                    'right': true,
+                    'items': buttons_action
                 } %}
             </div>
         </td>
     </tr>
-  {% else %}
-    <tr><td colspan="11">
+    {% endblock %}
+{% else %}
+<tr><td colspan="11">
```

Comment:
Weird indent

Reply:
was here previously, improving it now may make the contribution unreadable